app/javascript/packages/address-search/types.d.ts

```diff
@@ -0,0 +1,72 @@
+import type { RegisterFieldCallback } from '@18f/identity-form-steps';
+
```

Comment:
What do you think about alphabetizing all of the lists in this file?

Reply:
No. It is not necessary but I find it helpful, feel free to ignore if you don't like the idea

Reply:
If you use VS Code, the Alphabetize extension is a good one